KeenClient/KeenClient.m

```diff
@@ -224,7 +221,7 @@ - (id)initWithProjectId:(NSString *)projectId andWriteKey:(NSString *)writeKey a
             self.readKey = readKey;
         }
     }
-
+    
```

Comment:
Remove this space. Same for the others.

Reply:
Actually, that's the default behavior of Xcode, so anytime we reindent, Xcode will reproduce these space. I would rather avoid manually fixing Xcode stuff on this 😢